modules/distr-tracing-otel-migrating-from-jaeger-without-sidecars.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * distr-tracing-otel-migrating.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-otel-migrating-from-jaeger-without-sidecars_{context}"]
+= Migrating from the {JaegerShortName} to the {OTELShortName} without sidecars
+
+You can migrate from the {JaegerShortName} to the {OTELShortName} without sidecar deployment.
+
+.Prerequisites
+
+* The {JaegerName} is used on the cluster.
+* The {OTELName} is installed.
+
+.Procedure
+
+. Configure OpenTelemetry Collector deployment.
+
+. Create the project where the OpenTelemetry Collector will be deployed.
++
+[source,yaml]
+----
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: observability
+----
+
+. Create a service account for running the OpenTelemetry Collector instance.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-deployment
+  namespace: observability
+----
+
+. Create a cluster role for setting the required permissions for the processors.
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector
+rules:
+# Permissions for pods and namespaces resources are needed for the k8sattributesprocessor
+# Permissions for infrastructures and infrastructures/status are needed for resourcedetectionprocessor
+- apiGroups: ["", "config.openshift.io"]
+  resources: ["pods", "namespaces", "infrastructures", "infrastructures/status"]
+  verbs: ["get", "watch", "list"]
+----
+
+. Create a `ClusterRoleBinding` to set the permissions for the service account.
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: otel-collector
+subjects:
+- kind: ServiceAccount
+  name: otel-collector-deployment
+  namespace: observability
+roleRef:
+  kind: ClusterRole
+  name: otel-collector
+  apiGroup: rbac.authorization.k8s.io
+----
+
+. Create the OpenTelemetry Collector instance.
++
+NOTE: This collector will be exporting traces to a TempoStack instance. You must create your TempoStack instance using the Red Hat Tempo Operator and place here the correct endpoint.
```

Comment:
Probably can't avoid future tense here, but could use a simpler construction
```suggestion
NOTE: This collector will export traces to a TempoStack instance. You must create your TempoStack instance using the Red Hat Tempo Operator and place here the correct endpoint.
```

Reply:
AFAIK, `will be exporting` refers to a recurring activity rather than a single event.

Reply:
Thank you. Applied manually.